src/core/crypto/secp256k1.ts

```diff
@@ -53,7 +57,7 @@ export class Secp256k1PublicKey extends PublicKey {
    * @param args.message - The message that was signed.
    * @param args.signature - The signature to verify against the public key.
    */
-  verifySignature(args: VerifySignatureArgs): boolean {
+  verifySignature(args: { message: HexInput; signature: Secp256k1Signature }): boolean {
```

Comment:
isn't this a breaking change?

Reply:
I guess this is a breaking change technically...  though its a break that alerts the dev to code issues

but I'll revert it and deal with in another PR, maybe.